locale/locales/en.json

```diff
@@ -0,0 +1,10 @@
+{
+    "RowsPerPage": {
```

Comment:
Should we have better keys indicating that these are for the table?

Reply:
yea, they all get registred together so needs namespaces

Reply:
I see. Thanks. gonna fix

Reply:
fixed